egs2/binaural_librispeech/enh1/local/data.sh

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Aalborg University (Authors: Holger Severin Bovbjerg)
+# Apache 2.0
+set -e
+set -u
+set -o pipefail
+
+log() {
+  local fname=${BASH_SOURCE[1]##*/}
+  echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+SECONDS=0
+
+help_message=$(cat << EOF
+Usage: $0 [--min_or_max <min/max>] [--sample_rate <8k/16k>]
+  optional argument:
+    [--min_or_max]: min (Default), max
+    [--sample_rate]: 8k (Default), 16k
+EOF
+)
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+cdir=$PWD
+
+# Path to the directory containing MUSAN noise
+musan=${MUSAN}
+
+# Path to the directory containing Binaural LibriSpeech
+binaural_librispeech="${BINAURAL_LIBRISPEECH}"
+binaural_librispeech_subset="horizontal_plane_front_only"
+
+# Subsets to include in data preparation
+subsets=("train-clean-100" "train-clean-360" "train-other-500" "dev-clean" "dev-other" "test-clean" "test-other")
+
+min_or_max=max
+sample_rate=16k
+num_spk=2
+
+stage=0
+stop_stage=100
+
```

Comment:
You can add a variable `nj=4` here and use it to replace the hard-coded value `4` below.